skflow/__init__.py

```diff
@@ -51,11 +51,12 @@ class TensorFlowEstimator(BaseEstimator):
 
     def __init__(self, model_fn, n_classes, tf_master="", batch_size=32, steps=50, optimizer="SGD",
                  learning_rate=0.1, tf_random_seed=42, continue_training=False,
-                 log_device_placement=True):
+                 log_device_placement=True, verbose=1):
```

Comment:
(My previous comment is deleted - was probably too tired) 
@ilblackdragon Yes we definitely need to consolidate the output from both (`verbose` and `log_device_placement`) and maybe `print_steps`. Three parameters can be combined.
@ivallesp You'll have to resolve the conflict (rebase from upstream master). It would be better if you could consolidate the output too.